tslint.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "typestrict",
   "rules": {
-    "no-console": true,
+    "no-console": false,
```

Comment:
Could you revert this change? This is a useful rule, it's just typechain should encapsulate logging in one place but now it's everywhere :/